mmdet/models/backbones/resnet.py

```diff
@@ -356,6 +390,8 @@ def __init__(self,
                  stage_with_dcn=(False, False, False, False),
                  gcb=None,
                  stage_with_gcb=(False, False, False, False),
+                 gen_attention=None,
+                 stage_with_gen_attention=[[], [], [], []],
```

Comment:
It is better use tuple instead of list (or mutable types) for default arguments.

Reply:
Done.